internal/chatgpt/chatgpt.go

```diff
@@ -0,0 +1,70 @@
+package chatgpt
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/PullRequestInc/go-gpt3"
+	"github.com/slack-go/slack"
+	"github.com/xplorer-io/xplorers-bot-go/internal/config"
+	islack "github.com/xplorer-io/xplorers-bot-go/internal/slack"
+	"github.com/xplorer-io/xplorers-bot-go/internal/ssm"
+)
+
+type ChatGpt struct {
```

Comment:
`ChatGPT` all abbreviations should be capitalised. `mixedCaps` / `MixedCaps` are different

Reply:
Updated